modules/300-prometheus/crds/prometheusremotewrite.yaml

```diff
@@ -49,6 +49,12 @@ spec:
                   - password
                   - username
                   type: object
+                bearerToken:
+                  type: string
+                  description: Bearer token.
+                customAuthToken:
```

Comment:
Why not xAuthToken?

Reply:
Thought that we may change the exact header in the future. Using header name is kind of an exposing  implementation details, which I want to avoid.